src/store/types.ts

```diff
@@ -216,3 +216,7 @@ export enum EComputer {
   NORMAL = 1,
   INSANE = 2,
 }
+
+export type Season = {
+  id: number;
+};
```

Comment:
```suggestion
```

This type already exists in `src\store\ranking\types.ts`

Reply:
fixed